tests/integration/NewLexemeForm.test.ts

```diff
@@ -90,13 +90,17 @@ describe( 'NewLexemeForm', () => {
 		await languageInput.setValue( '=Q123' );
 		await wrapper.find( '.wbl-snl-language-lookup .wikit-OptionsMenu__item' ).trigger( 'click' );
 
+		const spellingVariantInput = wrapper.find( '.wbl-snl-spelling-variant-lookup input' );
+		await spellingVariantInput.setValue( 'en' );
```

Comment:
Maybe a comment here would be helpful? Something like `// will actually select en-gb, en is not in wikibaseLexemeTermLanguages`

Reply:
Done.